SuperNinjaCodingMasters/BikeShare-Project
# MongoDB Seeding Functional with Models

#### What

Created models for `Network` and `Station`, database seeding file allows these to be stored.

#### Changes

* `src/dbseed.js`

Modified general file structure, unsure of exact setup for React front end and whether a second `package.json` is needed or can be combined with the top level version in pathing. Created seeding script that works with the given files for networks and stations. Also changed all `CRLF` line endings back to `LF`.

#### Direction

We will now need to set up API routes to interface with the database in the final structure, and begin using the live data to display points in the front end application.

#### How to test

For just seeding the database, after setting up MongoDB on `localhost` port 27017 (will incorporate an `env` configuration going forward) and having it running, run `node --harmony ./src/dbseed.js` from the primary diectory. NOTE that currently if you run the script multiple times, the database will be seeded with duplicates.
